src/VirtoCommerce.CustomerExportImportModule.Data/Services/CsvCustomerDataValidator.cs

```diff
@@ -100,5 +104,80 @@ private void ValidateLineLimit(Stream stream, Configuration csvConfiguration, Li
                 errorsList.Add(error);
             }
         }
+
+        private static void ValidateRequiredColumns(Stream stream, Configuration csvConfiguration, List<ImportDataValidationError> errorsList)
+        {
+            var notCompatibleErrors = new[]
+            {
+                ModuleConstants.ValidationErrors.FileNotExisted,
+                ModuleConstants.ValidationErrors.ExceedingFileMaxSize,
+                ModuleConstants.ValidationErrors.WrongDelimiter,
+                ModuleConstants.ValidationErrors.NoData,
+            };
+
+            if (errorsList.Any(x => notCompatibleErrors.Contains(x.ErrorCode)))
+            {
+                return;
+            }
+
+            stream.Seek(0, SeekOrigin.Begin);
+            var streamReader = new StreamReader(stream);
+            var csvReader = new CsvReader(streamReader, csvConfiguration);
+
+            csvReader.Read();
+            csvReader.ReadHeader();
+
+            var existedColumns = csvReader.Context.HeaderRecord;
+
+            var requiredColumns = CsvCustomerImportHelper.GetImportPriceRequiredColumns();
+
+            var missedColumns = requiredColumns.Except(existedColumns).ToArray();
```

Comment:
comparing should be case insensitive. Please add the second parameter,  **StringComparer.InvariantCultureIgnoreCase**

Reply:
done